package-lock.json

```diff
@@ -2566,7 +2566,7 @@
         },
         "inquirer": {
           "version": "5.2.0",
-          "resolved": "http://registry.npmjs.org/inquirer/-/inquirer-5.2.0.tgz",
+          "resolved": "https://registry.npmjs.org/inquirer/-/inquirer-5.2.0.tgz",
```

Comment:
Are these change by npm on update?

Reply:
i do get that thing going every time I run npm install, don't know what causes it..

Reply:
In that case should we update master as well?